app/presenters/publishing_api_presenters.rb

```diff
@@ -27,6 +18,12 @@ def self.presenter_class_for(model)
       PublishingApiPresenters::TakePart
     when Topic
       PublishingApiPresenters::PolicyAreaPlaceholder
+    when ::StatisticsAnnouncement
+      if model.unpublished? || (model.publication && model.publication.published?)
```

Comment:
I think the `StatisticsAnnouncement` should be the canonical source for this. This random module (random against the backdrop of `StatisticAnnouncement` responsibilities) violating demeter doesn't feel right. Something like `requires_redirect?` feels better, I guess it's a State of `StatisticsAnnouncement` that should be captured/named properly.